tests/e2e/e2e_rest_regression_test.go

```diff
@@ -0,0 +1,89 @@
+package e2e
+
+import (
+	"fmt"
+	"net/http"
+)
+
+/*
+RestRegression tests the continuity of critical endpoints that node operators, block explorers, and ecosystem participants depend on.
+Test Node REST Endpoints:
+1. http://host:1317/validatorsets/latest
+2. http://host:1317/validatorsets/{height}
+3. http://host:1317/blocks/latest
+4. http://host:1317/blocks/{height}
+5. http://host:1317/syncing
+6. http://host:1317/node_info
+7. http://host:1317/txs
+Test Module REST Endpoints
+1. Bank total
+2. Auth params
+3. Distribution for Community Pool
+4. Evidence
+5. Gov proposals
+6. Mint params
+7. Slashing params
+8. Staking params
+*/
+
+var (
```

Comment:
Nit: maybe consider using `const`.
```suggestion
const (
```

Reply:
Fixed nit